src/api/mas/GetMas.ts

```
@@ -0,0 +1,33 @@
+/* Import Dependencies */
+import axios from "axios";
+
+/* Import Types */
+import { MachineAnnotationService } from "app/types/MachineAnnotationService";
+import { JSONResult } from "app/Types";
+
+
+const GetMas = async (masId: string) => {
+    let mas: MachineAnnotationService | undefined;
+
+    if (masId) {
```

what happens `if not (masId)`? Should you throw an error or log?

---

If it fails, it will throw an error in the console and return an undefined, which is handled in the file which called on this function.
Agree that the error handling is certainly not on pal with DiSSCover, or indicating load times etc.